cwm_worker_operator/deployer.py

```diff
@@ -217,12 +222,30 @@ def get_deployment_config(debug, domains_config, extra_minio_extra_configs, name
     return deployment_config, extra_objects
 
 
-def run_single_iteration(domains_config: domains_config_module.DomainsConfig, metrics, deployments_manager, extra_minio_extra_configs=None, **_):
-    deployer_metrics = metrics
+class DeployerMultiprocessor(Multiprocessor):
+
+    def _run_async(self, worker_id, extra_minio_extra_configs, domains_config, metrics, deployments_manager, flow_manager):
+        cmd = ['cwm-worker-operator', 'deployer', 'deploy_worker', '--worker-id', worker_id]
+        if extra_minio_extra_configs:
+            cmd += ['--extra-minio-extra-configs', json.dumps(extra_minio_extra_configs)]
+        return subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
Shouldn't the `Multiprocessor` itself take the responsibility of creating the `subprocess.Popen`?

Reply:
I want to provide flexibility in creating it, maybe other processes will want to create if differently, do other actions before / after creating the process